### PR TITLE
fix(karma): Load karma-* dependencies from @bazel/karma

### PR DESCRIPTION
**Pull request by kyliau**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/karma/plugins.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = [
```
Comment (alexeagle) [inline packages/karma/plugins.js]: could use a comment explaining why we pick these plugins, and how this file ensures we get the right version

File: packages/karma/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "karma": "^4.0.0",
     "karma-chrome-launcher": "2.2.0",
     "karma-firefox-launcher": "1.1.0",
-    "karma-jasmine": "1.1.1",
+    "karma-jasmine": "2.0.1",
```
Comment (alexeagle) [inline packages/karma/package.json]: is this version bump required in this PR? just curious

Comment (kyliau) [reply packages/karma/package.json]: No, this version bump is not required in this PR. I've reverted this change, will create another PR just for this.

File: packages/karma/plugins.js (hunk 1 above)
Comment (kyliau) [reply packages/karma/plugins.js]: done.

**Pull request merged**